CMakeLists.txt

```diff
@@ -14,6 +14,10 @@ read_version("${CMAKE_CURRENT_SOURCE_DIR}/version.pri" VER_MAJOR VER_MINOR VER_B
 
 project(qBittorrent VERSION ${VER_MAJOR}.${VER_MINOR}.${VER_BUGFIX}.${VER_BUILD})
 
+# check for invalid compiler version/CXX standard as early as possible
+include(FunctionQbtCXXCompilerAndModeCheck)
+qbt_cxx_compiler_and_mode_check("14")
```

Comment:
Why is argument passed as string?

Reply:
Well everything is a string in CMake anyway. But, it is better to unquote it I guess, since elsewhere I also use `98` without quotes. Fixed.